README.md

```diff
@@ -331,9 +331,38 @@ Please use GitHub's Issues/PRs.
 
 ### Development Tools
 
-- `yarn test` runs tests and measures coverage.
+- `yarn test` runs tests.
+- `yarn cover` runs tests and measures coverage.
 - `yarn update` runs in order to update readme and recommended configuration.
 
+### Test the Rule
+
+Rule testing almost always uses fixtures.  
+For example, for an `indent` rule, the `.ts` file that runs the test is `tests/src/rules/indent.ts` and the fixture is in `tests/fixtures/rules/indent`.  
+The fixture directory has an `invalid` directory and a `valid` directory.
+
+- The `invalid` directory contains test cases where the rule reports problems.
+- The `valid` directory contains test cases where the rule does not report a problem.
+
+The fixture input file should be named `*-input.svelte`. It is automatically collected and tested.  
+If your test requires configuration, you need to add a json file with the configuration.
+
+- If you want to apply a configuration to `my-test-input.svelte`, add `my-test-config.json`.
+- If you want to apply the same configuration to all the fixtures in that directory, add `_config.json`.
+
+To verify the output of invalid test cases requires `*-errors.json`, and `*-output.svelte` (for auto-fix). However, you don't have to add them yourself. If they do not exist, they will be automatically generated when you run the test. In other words, delete them manually when you want to recreate them.
+
+**Tips**:
+
+If you want to test only one rule, run the following command (for `indent` rule):
+
+```sh
+yarn mocha "tests/src/**/indent.ts" --reporter dot --timeout 60000
```

Comment:
Should `-g, --grep <pattern>            only run tests matching <pattern>` be preferred?

https://stackoverflow.com/questions/10832031/how-to-run-a-single-test-with-mocha

Reply:
The StackOverflow url could be added as reference.

Reply:
I find it difficult to use `-g` because it's hard for the ESLint rule tester to find out what to register in `it()`.

Reply:
😅Fair enough.

Reply:
I misunderstood what you were saying.
It means to narrow down the test by the rule name. For example, `-g indent`.
I think it's easy to use, so I will change it to use `-g`.